packages/api/models/mappings.json

```diff
@@ -65,6 +65,42 @@
       }
     }
   },
+  "async_operation": {
```

Comment:
This should be camelcased instead of using the `_`. In other places the you specify the type as `asyncOperation`, so this should match.